<launch>
    <arg name="max_vel" value="0.5" />
    <arg name="max_acc" value="0.5" />
    <arg name="dynamic" value="0" />

        <!-- trajectory server -->
    <node pkg="prometheus_local_planning" name="local_planner_main" type="local_planner_main" output="screen">
        <!-- <remap from="/odom_world" to="/state_ukf/odom"/>
        <remap from="/position_cmd" to="planning/pos_cmd"/> -->
        <param name="local_planning/is_simulation" value="1" type="int"/>
        <param name="apf/obs_distance" value="0.8" type="double"/>
        <param name="apf/k_push" value="1.5" type="double"/>
    </node>
        
    <!-- use simulator -->
    <include file="$(find prometheus_local_planning)/launch/simulator.xml">
    </include>

</launch>